utils/.gitignore

```diff
@@ -1,8 +0,0 @@
-!*.ipynb
```

Comment:
our `utils` folder seemed both old and untested, so I'm recommending we delete it for now. we can always bring it back in a separate repo should we want to do performance / memory profiling in the future